card/mod/history/set/all/content_history.rb

```diff
@@ -164,5 +164,6 @@ def draft_acts
 def edit_conflict?
   last_action_id_before_edit &&
     last_action_id_before_edit.to_i != last_action_id &&
-    last_action.act.actor_id != Auth.current_id
+    (la = last_action) &&
+    la.act.actor_id != Auth.current_id
 end
```

Comment:
Maybe a session card gets an action if it is created but not when it is updated? just a guess